src/components/pages/admin/events/dashboard/fanNotifications/Index.js

```diff
@@ -280,7 +299,7 @@ class Index extends Component {
 			: this.createNotification(broadcastData);
 	}
 
-	createNotification(broadcastData) {
+	async createNotification(broadcastData) {
 		Bigneon()
```

Comment:
Wrap this in a try and add await
```suggestion
		await Bigneon()
```